main.py

```diff
@@ -1,7 +1,53 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
+
+
+genre_items = (
```

Comment:
Why not call it genre_names and actor_info or something more appropriate then items?
Also these are used as global variables in your code which is not a good practice - you either declare them inside main or pass to main() as arguments